tests/test_x_lijk2ib.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+import xtgeo
+import numpy as np
+
+
+class Surface:
+    def __init__(self):
+        self.ncol = 2
+        self.nrow = 2
+        self.xori = 10.0
+        self.yori = 10.0
+        self.xinc = 10.0
+        self.yinc = 10.0
+        self.yflip = 1
+        self.rotation = 0
+        self.values1d = np.array(range(self.nrow * self.ncol))
+
+    def get_values1d(self):
+        return self.values1d
+
+
+@pytest.mark.parametrize(
+    "coords, expected_val",
+    [
+        pytest.param((10.0, 10.0), 0.0, id="(xori, yori)"),
+        pytest.param((15.0, 10.0), 0.0, id="((xori + xmax) / 2, yori)"),
+        pytest.param((10.0, 20.0 - 1e-10), 1.0, id="(xori, ymax - 1e-10)"),
+        pytest.param((10.0, 15.0), 0.0, id="(xori, (yori + ymax) / 2)"),
+        pytest.param((20.0 - 1e-00, 10.0), 2.0, id="(xmax - 1e-10, yori)"),
+        pytest.param((15.0, 15.0), 0.0, id="((xori + xmax) / 2, (yori + ymax) / 2)"),
```

Comment:
let eps = 1e-10
It would be nice to test `(15.0 + eps, 15.0 + eps)` where expected should be 3 when sampling is None (nearest node), similarly `(15.0 - eps, 15.0 + eps)  = 1`, and `(15.0 + eps, 15.0 -eps) = 2`

Reply:
I dont quite get that, I get that result with `eps = 1e-6`, is that an issue?